zerver/lib/upload.py

```diff
@@ -181,13 +188,27 @@ def resize_emoji(image_data: bytes, size: int = DEFAULT_EMOJI_SIZE) -> bytes:
                 or im.size[0] > MAX_EMOJI_GIF_SIZE  # dimensions too large
                 or len(image_data) > MAX_EMOJI_GIF_FILE_SIZE_BYTES  # filesize too large
             )
-            return resize_gif(im, size) if should_resize else image_data
+
+            # Generating still image.
+            im.seek(1)
```

Comment:
This should be `im.seek(0)`, for the first frame, right?

Reply:
Changed this with my push; @Riken-Shah please confirm whether you agree this change is correct.

Reply:
Yeah, this seems right!